hw/ip/clkmgr/data/clkmgr_testplan.hjson

```diff
@@ -139,5 +182,21 @@
             but the dvsim coverage flow doesn't yet support arrays.
             '''
     }
+    {
+      name: extclk_cg
+      desc: '''
+            Collects coverage for the external clock selection.
+
+            The external clock selection depends on the `extclk_sel` CSR,
+            and the `lc_dft_en_i`, `lc_clk_byp_req_i`, and `scanmode_i`
+            input pins. This covergroup collects their cross.
+            '''
+    }
+    {
+      name: jitter_cg
+      desc: '''
+            Collects coverage of the `jitter` CSR.
+            '''
+    }
```

Comment:
This should show up as toggle on clkmgr IOs, right?

Reply:
This intends to collect coverage on the CSR. The jitter_en_o output pin will show up in toggle coverage, and the test checks their causal relation. Seems a bit redundant, and I would not mind removing this.

By the way, do we automagically collect coverage on all CSRs?

Reply:
Okay, I'm fine either way.